samples/dotnet/clientcreds/Lenus.Samples.ClientCredentialsFlow/src/Configuration/ConfigureAgencyOptions.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Lenus.Samples.ClientCredentialsFlow.Configuration
+{
+    using Microsoft.Extensions.Options;
```

Comment:
Some weird formatting going on here causing usings to be within the namespace. This doesn't seem to be the case for other projects in this repo?

Reply:
I have moved all of the using statements outside the namespace